templates/zerver/app/compose.html

```diff
@@ -97,8 +97,12 @@
                                     <input type="file" id="file_input" class="notvisible pull-left" multiple />
                                     <a class="message-control-button fa fa-smile-o" aria-hidden="true" id="emoji_map" href="#" title="{{ _('Add emoji') }}"></a>
                                     <a class="message-control-button fa fa-font" aria-hidden="true" title="{{ _('Formatting') }}" data-overlay-trigger="message-formatting"></a>
-                                    <a class="message-control-button fa fa-paperclip notdisplayed" aria-hidden="true" id="attach_files" href="#" title="{{ _('Attach files') }}"></a> {% if jitsi_server_url %}
-                                    <a class="message-control-button fa fa-video-camera video_link" aria-hidden="true" href="#" title="{{ _('Add video call') }}"></a> {% endif %}
+                                    {% if max_file_upload_size > 0 %}
+                                    <a class="message-control-button fa fa-paperclip notdisplayed" aria-hidden="true" id="attach_files" href="#" title="{{ _('Attach files') }}"></a>
+                                    {% endif %}
+                                    {% if jitsi_server_url %}
+                                    <a class="message-control-button fa fa-video-camera video_link" aria-hidden="true" href="#" title="{{ _('Add video call') }}"></a>
+                                    {% endif %}
```

Comment:
@skunkmb quick question for you: Why is this conditional `if jitsi_server_url`?  I would think that wouldn't work with other configured video chat providers...